core/services/ocr2/validate/config.go

```diff
@@ -45,10 +48,31 @@ func ToLocalConfig(ocr2Config OCR2Config, insConf InsecureConfig, spec job.OCR2O
 		ContractTransmitterTransmitTimeout: ocr2Config.ContractTransmitterTransmitTimeout(),
 		DatabaseTimeout:                    ocr2Config.DatabaseTimeout(),
 	}
+	if spec.Relay == relay.Solana && env.SolanaPluginCmd.Get() != "" {
+		// Work around for Solana LOOPPs configured with zero values.
+		lc.MinOCR2MaxDurationQuery = getMinOCR2MaxDurationQuery()
+	}
 	if insConf.OCRDevelopmentMode() {
 		// Skips config validation so we can use any config parameters we want.
 		// For example to lower contractConfigTrackerPollInterval to speed up tests.
 		lc.DevelopmentMode = types.EnableDangerousDevelopmentMode
 	}
 	return lc
 }
+
+var (
+	minOCR2MaxDurationQuery     = 20 * time.Millisecond
+	minOCR2MaxDurationQueryOnce sync.Once
+)
+
+func getMinOCR2MaxDurationQuery() time.Duration {
+	minOCR2MaxDurationQueryOnce.Do(func() {
+		if v := env.MinOCR2MaxDurationQuery.Get(); v != "" {
+			d, err := time.ParseDuration(v)
```

Comment:
we should return and handle this error. it will be mystical behavior if the env var doesn't parse

Reply:
Oh it may not be so bad actually - update incoming